binding.gyp

```diff
@@ -57,7 +57,7 @@
 
   "targets": [
     {
-      "target_name": "agentcore",
+      "target_name": "omr-agentcore",
```

Comment:
Agreed - it's needed to build from source locally on Windows. I'll move them to a new pull request.